internal/network/utils.go

```diff
@@ -239,3 +239,31 @@ func GetIPv4Networks(inventory *models.Inventory) (ret []string, err error) {
 func GetIPv6Networks(inventory *models.Inventory) (ret []string, err error) {
 	return getHostNetworks(inventory, func(i *models.Interface) []string { return i.IPV6Addresses })
 }
+
+func areNetworksIdentical(len1, len2 int, f func(int, int) bool) bool {
+	if len1 != len2 {
+		return false
+	}
+next:
```

Comment:
Is this `next:` desired here?

Reply:
Ok, after googling a bit I'm not sure there is more elegant way to escape from double for-loop (another option being extracting to a yet separate function)

Reply:
I will be happy for suggestions.
The only way I am aware of is to put the inner loop in a function

Reply:
I think putting the inner loop in a named (local-to-this-function) function would be helpful. I don't think the labels are used frequently enough for people to be super familiar with them.